Makefile

```diff
@@ -19,11 +19,27 @@ tmp/dev_image_id:
 format: prepare
 	${DOCKRUN} bash ./scripts/format.sh
 
-check: prepare
+check: prepare format
 	${DOCKRUN} bash ./scripts/check.sh
 
-todo: prepare
+todo:
 	${DOCKRUN} bash ./scripts/todo.sh
 
-test: prepare format check
+test: check db_seed
 	${DOCKRUN} bash ./scripts/test.sh
+
+db_start: db_stop
+	@docker run --name unit-testing-mysql-db -e MYSQL_ALLOW_EMPTY_PASSWORD=yes -p 3306:3306 -d mysql:5.6
```

Comment:
How about we name these containers:
`apna-school-mysql-db`
`apna-school-mongo-db`